includes/class-wc-beta-tester-admin-assets.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Beta Tester Admin Assets class
+ *
+ * @package WC_Beta_Tester
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Beta_Tester_Admin_Assets Class.
+ */
+class WC_Beta_Tester_Admin_Assets {
+
+	/**
+	 * Hook in tabs.
+	 */
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_scripts' ) );
+	}
+
+	/**
+	 * Enqueue scripts.
+	 */
+	public function admin_scripts() {
+		$suffix       = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+		// Need admin styles for the modal.
+		wp_enqueue_style( 'woocommerce_admin_styles' );
```

Comment:
Do we need to include all WC styles here? Can we include modal only styles instead?

Reply:
Seems `admin.css` contains all the data necessary for modals, and `woocommerce_admin_styles` includes precisely it.